library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -658,7 +661,7 @@ private void setLastDateSelectable(Date start) {
       for (List<MonthCellDescriptor> week : month) {
         for (MonthCellDescriptor singleCell : week) {
           if (singleCell.getDate().after(start)
-              && !singleCell.isSelectable()
+              && !isDateSelectable(singleCell.getDate())
```

Comment:
Testing with the ``isDateSelectable ``  function just in case the cell got unlocked